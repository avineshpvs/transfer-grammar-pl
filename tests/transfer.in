1	((	NP	<fs af='rAma,n,,,,o,adj_vAdu,' head="rAmudu">
1.1	rAmudu	NNP	<fs af='rAma,n,,,,o,adj_vAdu,' name="rAmudu">
	))		
2	((	NP	<fs af='mohanvalYlu,unk,,,,,,' head="mohanvalYlu">
2.1	mohanvalYlu	NN	<fs af='mohanvalYlu,unk,,,,,,' name="mohanvalYlu">
	))		
3	((	NP	<fs af='pani,n,,s,,0,0_V,' head="panI">
3.1	panI	NN	<fs af='pani,n,,s,,0,0_V,' name="panI">
	))		
4	((	VGF	<fs af='ceVyyi,v,,,23_,0,wunn_ani,' head="ceswunnArani">
4.1	ceswunnArani	VRB	<fs af='ceVyyi,v,,,23_,0,wunn_ani,' name="ceswunnArani">
	))		
5	((	VGF	<fs af='ceppedu,v,,,,,,' head="ceppedu">
5.1	ceppedu	VFM	<fs af='ceppedu,v,,,,,,' name="ceppedu">
5.2	.	SYM	
	))		
