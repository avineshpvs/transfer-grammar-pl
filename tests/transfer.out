1	((	NP	<fs af='rAma,n,,,,o,adj_vAdu,' head="rAmudu">
1.1	rAmudu	NNP	<fs af='rAma,n,,,,o,adj_vAdu,' name="rAmudu">
	))		
2	((	NP	<fs af='mohanvalYlu,unk,,,,,,' head="mohanvalYlu">
2.1	mohanvalYlu	NN	<fs af='mohanvalYlu,unk,,,,,,' name="mohanvalYlu">
	))		
3	((	NP	<fs af='pani,n,,s,,0,0_V,' head="panI">
3.1	panI	NN	<fs af='pani,n,,s,,0,0_V,' name="panI">
	))		
4	((	VGF	<fs af='ceVyyi,v,,,23_,0,wunn,' head="ceswunnArani">
4.1	ceswunnArani	VRB	<fs af='ceVyyi,v,,,23_,0,wunn,' name="ceswunnArani">
	))		
5	((	NP	<fs af='ani,psp,,,,,,' head="ani">
5.1	ani	PSP	<fs af='ani,psp,,,,,,' name="ani">
	))		
6	((	VGF	<fs af='ceppedu,v,,,,,,' head="ceppedu">
6.1	ceppedu	VFM	<fs af='ceppedu,v,,,,,,' name="ceppedu">
6.2	.	SYM	
	))		
